solution/ui/regulations/eregs-component-lib/src/components/SupplementalContentList.vue

```diff
@@ -1,14 +1,14 @@
 <template>
     <div class="supplemental-content-list" v-if="!has_sub_categories">
-        <template
-            v-for="(content, index) in limitedContent"
-        >
+        <template v-for="(content, index) in limitedContent">
             <supplemental-content-object
                 :key="index"
                 :name="content.name"
-                :description="content.description"
-                :date="content.date"
+                :description="content.description || content.doc_name_string"
+                :date="content.date || content.date_string"
```

Comment:
allow existing `supplemental-content-object` component to be used with external resource item or internal document item